### DISCUSSION TITLE
Is there a way to add a new space with a key-command?

### DISCUSSION THREAD

**Discussion by DCZare** [ianyh/Amethyst#1356] [Ideas]
I am having a lot of fun with Amethyst and I find myself using the mouse less and less, however, having to use a mouse to add and remove spaces is kind of a buzzkill when I'm in a flow.

That being said, if it's not a feature due to some limitations with MacOS or just having never gotten to it I'd understand.

Thank you so much for your time, 
I appreciate you folks and hope y'all keep being beautiful developer wizards

**Comment by ianyh** [comment]
Not implemented, but I think possible. #143

**Reply by DCZare** [reply]
I think both the ability to make new screens and a toggle to allow you to automatically create new spaces if you throw left or right to a space but there are no more spaces in that direction. That being said, I can see such a recommendation such as a toggle can be an overzealous request. Overall, consider this a wishlist for a dream build from a barely CS literate tmux and lunarvim user trying to use the mouse less and less so you can take everything with a grain of salt.


Rearranging and adding spaces is always the biggest interruption to my keyboard flow so as much as amethyst has been an amazing experience I hope there is enough interest for these implementations to speed up that flow even more
